public/svg/OpenIcon.tsx

```diff
@@ -1,18 +1,18 @@
 export function OpenIcon() {
   return (
     <svg
-      width="18"
-      height="12"
-      viewBox="0 0 18 12"
+      width="28"
+      height="28"
+      viewBox="0 0 28 28"
       fill="none"
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        d="M18 6C18 6.3608 16.6089 7.85641 14.9205 9.53874C11.627 12.8204 6.37298 12.8204 3.0795 9.53875C1.39111 7.85641 0 6.3608 0 6C0 5.6392 1.39111 4.14359 3.0795 2.46126C6.37298 -0.820417 11.627 -0.820419 14.9205 2.46125C16.6089 4.14359 18 5.6392 18 6Z"
-        fill="#a6a6a6"
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```
jsx(tsx)에서는 속성을 쓸 때 카멜케이스 표기법을 사용합니다